sources/packages/backend/apps/api/src/app.aest.module.ts

```diff
@@ -31,6 +31,7 @@ import {
   ApplicationOfferingChangeRequestService,
   StudentLoanBalanceService,
   ProgramYearService,
+  AnnouncementsService,
```

Comment:
https://github.com/bcgov/SIMS/pull/3678/files#r1747611024

Please remove adding the provider here.

Reply:
removed